src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.spec.ts

```diff
@@ -36,4 +36,170 @@ describe('InstructorCoursesPageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with all courses in course stats', () => {
+    const activeCourses: any[] = [
+      {
```

Comment:
Just as a suggestion, maybe you can extract out the course objects as a variable outside of the test case to prevent duplication. Becos for example, there are another 2 instances of this exact same object. Same goes for the other course or courseStat objects